helpers/build_packages.sh

```diff
@@ -152,7 +152,7 @@ buildmw qt5-feedback-haptics-droid-vibrator || die
 buildmw qt5-qpa-hwcomposer-plugin || die
```

Comment:
nitpick: please remove the full stop in the commit msg, so it doesn't look like an unfinished release number 2.1.1.xyz

Reply:
Commit message changes.